common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -536,6 +535,57 @@ def test_no_content(self):
             "You have not made any notes in this course yet. Other students in this course are using notes to:",
             notes_page_empty.no_content_text)
 
+    def test_notes_works_correctly_with_xss(self):
+        """
+        Scenario: Note text & tags should be HTML and JS escaped
+        Given I am enrolled in a course with notes enabled
+        When I visit the Notes page, with a Notes text & tag containing JS code
+        Then I should see them escaped.
```

Comment:
Thanks for the offline conversation and detailing how this worked.

Can we change the comment to:
```
Given ...
When I visit the Notes page, with a Notes text and tag containing HTML characters like < and >
Then the tags appear as expected due to having been properly escaped
```